### PR TITLE
support new datapoint tags property

### PR DESCRIPTION
**Pull request by cluther**
Implements ZPS-4587.

### CODE REVIEW

**Comment by cluther** [comment]
retest

**Comment by cluther** [comment]
retest this please

**Comment by cluther** [comment]
retest this please

**Comment by cluther** [comment]
retest this please

File: ZenPacks/zenoss/PythonCollector/services/PythonConfig.py [hunk 1]
```diff
@@ -127,6 +127,12 @@ def _datasources(self, deviceOrComponent, deviceId, componentId, collector):
                     if hasattr(deviceOrComponent, 'getMetricMetadata'):
                         dp_config.metadata = deviceOrComponent.getMetricMetadata()
 
+                    # Support for RRDDataPoint.tags added in Zenoss 7.
+                    if hasattr(dp, "getTags"):
```
Comment (pcarinhas) [inline ZenPacks/zenoss/PythonCollector/services/PythonConfig.py]: Is it possible that getTags is missing but that tags would be present, thereby zeroing them out? IE. Some sort of out-of-order installation?

Comment (cluther) [reply ZenPacks/zenoss/PythonCollector/services/PythonConfig.py]: No. The tags and getTags attributes were added to RRDDataPoint in the same commit. Though even if it was possible, this is safe. The only user of dp_config.tags is zenpython, and it is sufficiently guarding its usage to be tolerant of no tags attribute on dp_config, a None value, an empty dict, or a dict full of tags.

**Pull request merged**